op_compatibility/first_wave_models.md

```diff
@@ -10,33 +10,33 @@
 | [NSNet](https://arxiv.org/pdf/2001.10601.pdf) | Noise Suppression |
 
 ## The ops required by the first-wave models
-| Op | SqueezeNetV1.1 [ONNX](https://github.com/onnx/models/tree/master/vision/classification/squeezenet) [TFLite](https://www.tensorflow.org/lite/guide/hosted_models) | MobileNetV2 [ONNX](https://github.com/onnx/models/tree/master/vision/classification/mobilenet) [TFLite](https://www.tensorflow.org/lite/guide/hosted_models) | ResNetV2 [ONNX](https://github.com/onnx/models/tree/master/vision/classification/resnet) [TFLite](https://www.tensorflow.org/lite/guide/hosted_models) | TinyYOLOV2 [ONNX](https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/tiny_yolov2) [TFLite](https://github.com/intel/webml-polyfill/tree/master/examples/object_detection/model#for-tiny-yolo-models) | RNNoise [C](https://github.com/xiph/rnnoise/blob/master/src/rnn.c) | NSNet [ONNX](https://github.com/microsoft/DNS-Challenge/tree/master/NSNet-baseline) | ONNX | XLA-HLO |
+| Op | SqueezeNetV1.1 [ONNX](https://github.com/onnx/models/tree/master/validated/vision/classification/squeezenet) [TFLite](https://www.tensorflow.org/lite/guide/hosted_models) | MobileNetV2 [ONNX](https://github.com/onnx/models/tree/master/validated/vision/classification/mobilenet) [TFLite](https://www.tensorflow.org/lite/guide/hosted_models) | ResNetV2 [ONNX](https://github.com/onnx/models/tree/master/validated/vision/classification/resnet) [TFLite](https://www.tensorflow.org/lite/guide/hosted_models) | TinyYOLOV2 [ONNX](https://github.com/onnx/models/tree/master/validated/vision/object_detection_segmentation/tiny-yolov2) [TFLite](https://github.com/intel/webml-polyfill/tree/master/examples/object_detection/model#for-tiny-yolo-models) | RNNoise [C](https://github.com/xiph/rnnoise/blob/master/src/rnn.c) | NSNet [ONNX](https://github.com/microsoft/DNS-Challenge/tree/2fe1bf1b13068e558169790e5443c5eba51a7901/NSNet-baseline) | ONNX | XLA-HLO |
```

Comment:
Is this NSNet-baseline reference is the latest? I do see the master branch no longer hosts this, so unless this has moved elsewhere that is probably the case.

(For NSNet2, we refer to https://github.com/microsoft/DNS-Challenge/tree/a052ad5a7714bcc6069b515666e837bf973099de/NSNet2-baseline in the [explainer](https://github.com/webmachinelearning/webnn/blob/main/explainer.md) for the same reason.)